company.json

```diff
@@ -7,15 +7,18 @@
-  "summary_en": "We are a startup focused on innovative IT projects in healthcare. We are able to deliver end to end products from the initial idea formation and business analysis to production deployment with full customer support. We build our solutions on AI, machine learning and optimization, but can easily handle web & mobile app development as well. A key factor of our work is a deep understanding of the medical aspect of our projects and perfect knowledge of the healthcare environment. We are a team of motivated people, ready to solve any interesting healthcare problem and help improve the medical world. We have several products ready for production deployment, but we are also able to create on-demand tailor-made solutions to fit your exact needs.",
-  "summary_de": "Wir sind ein Startup, das sich auf die Entwicklung innovativer IT-Lösungen im Gesundheitswesen orientiert. Wir können unsere Projekte vollständig von der Grundidee und Problemanalyse bis zum Einsetzen in Produktionsumgebung mit voller Kundenunterstützung liefern. Wir nutzen die künstliche Intelligenz, das maschinelle Lernen und die Optimierung in vollem Umfang, aber wir können auch die Entwicklung von Web- oder Mobile Apps problemlos besorgen. Ein Schlüsselfaktor ist auch unsere perfekte Kenntnis des Gesundheitswesens und die Fähigkeit, medizinische Probleme gut zu verstehen. Wir sind ein Team von begeisterten Menschen, die bereit sind, jedes interessante medizinische Problem zu lösen und die medizinische Welt zu verbessern. Derzeit arbeiten wir an mehreren solchen Projekten, von denen einige schon in Krankenhäusern eingesetzt sind.",
+  "summary_cs": "Držíme krok s nejmodernějšími technologiemi ale rozumíme specifikám zdravotnictví. Komunikujeme přímo se zdravotníky, abychom identifikovali nejaktuálnější výzvy. Řešení vytváříme iterativně. Zaměřujeme se na to, aby byl již po krátké době připraven první prototyp, který vyhodnocujeme v úzké spolupráci s odborným personálem zdravotnických zařízení. Každá iterace pak přináší hlubší porozumění problému i konkrétnější požadavky na vylepšení. Nejsme vázáni na konkrétní technologii, protože věříme tomu, že je důležité technologii přizpůsobit řešení a ne naopak. Typickým výsledkem pak je webová/mobilní aplikace či rozšíření již existujícího nástroje.",
```

Comment:
tady mi nesedi ze je to nazvany summary, kdyz to ma nadpsi how we a owrk. Asi bych tomu dal i takovy jmeno, at je to jasny. 

a teda zas mi ta prvni teta prijde ze nerika jak pracujeme. To mi prijde ze rikaji az ty dalsi. Jestli ta prvni veta ma shrnout jak pracujeme, tak to nedela.

to by bylo "Iterativne vyvijme zdravotnicke aplikace v uzke spolupraci se zdravotniky." Nebo tak neco

Reply:
Nazvy tych keys premenujem (aj mission je z tohto pohladu nie uplne vhodne). Pridam mission a vision ako samostatné entries, ktoré ale zatiaľ nebudú na webe prezentované. 

Po debate ponecham povodne znenie iba zmenim "ale" na "a zaroven"